ecommerce_app.py

```diff
@@ -0,0 +1,186 @@
+from user import User
+from product import Product
+
+
+MENU = '''
+    ---- Main Menu ----
+    
+    1. Login
+    2. Register
+    3. Add Product
+    4. Remove Product
+    5. Search Product
+    6. List Products
+    7. Exit 
+'''
+
+
+class ECommerce:
+    def __init__(self):
+        self.users = []
+        self.current_user = None
+        self.products = []
+    
+    def get_user_input(self, display_text, invalid_message, dtype = str):
```

Comment:
shift this function to a file called utils.py as this is a utility function and then import it